### PR TITLE
Update Trivy-Adapter README

### PR DESCRIPTION
**Pull request by adeniyistephen**
Fix: I updated the third step on using trivy-adapter from: "Create a vulnerability report customResourceDefinition" to "Create resources in the cluster that trivy-adapter will use"

### CODE REVIEW

File: policy-report/trivy-adapter/README.md [hunk 1]
```diff
@@ -49,4 +49,4 @@ kubectl get policyreports pod-zipkin-uewcde32cs9-ui -o yaml
 ```
 #### Road Maps:
```
Comment (JimBugwadia) [inline policy-report/trivy-adapter/README.md:50]: Change to "Roadmap"

**Review by JimBugwadia — CHANGES_REQUESTED** [review]

File: policy-report/trivy-adapter/README.md (hunk 1 above)
Comment (adeniyistephen) [reply policy-report/trivy-adapter/README.md]: Done.

**Review by JimBugwadia — APPROVED** [review]

**Comment by adeniyistephen** [comment]
LGTM

**Pull request merged**